docs-requirements.txt

```diff
@@ -15,6 +15,7 @@ opentracing~=2.2.0
 prometheus_client>=0.5.0,<1.0.0
 psycopg2-binary>=2.7.3.1
 pymongo~=3.1
+pyramid>=1.7
```

Comment:
is there no requirements for sqlite3?

Reply:
sqlite3 is included in base python (https://docs.python.org/3/library/sqlite3.html), so no

Reply:
🤦